tests/kernels/test_triton_flash_attention.py

```diff
@@ -0,0 +1,585 @@
+# SPDX-License-Identifier: Apache-2.0
+"""Tests for the triton_flash_attention kernel
+
+Run `pytest tests/kernels/test_triton_flash_attention.py`.
+"""
+import pytest
+import torch
+
+from vllm import _custom_ops as ops
+from vllm.attention.ops.triton_flash_attention import (SUPPORTED_LAYOUTS,
+                                                       MetaData,
+                                                       compute_alibi_tensor,
+                                                       triton_attention_rocm)
+from vllm.platforms import current_platform
+
+FP8_DTYPE_TORCH = current_platform.fp8_dtype()
+
+float8_info = torch.finfo(FP8_DTYPE_TORCH)
+FP8_MIN = float8_info.min
+FP8_MAX = float8_info.max
```

Comment:
From my experience, if you use `tl.constexpr`, you can use these global constants in kernels directly:
```suggestion
FP8_MIN: tl.constexpr = float8_info.min
FP8_MAX: tl.constexpr = float8_info.max
```

Reply:
Done